201-api-management-logs-to-moesif-using-eventhub-webapp/azuredeploy.json

```diff
@@ -41,6 +43,19 @@
                 "description": "A prefix that will be added to created resource names and DNS URLs. Allowed characters: alphabets and numbers only. Resulting name must be maximum 24 characters (storage account maximum)"
             }
         },
+        "azureWebsitesDomain": {
```

Comment:
use the environment() function to determine the cloud and set the appropriate suffix for the user - using a param will just create more opportunities for user error.

Reply:
I reviewed and re-ran the template on [Azure Template Functions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-deployment?tabs=json#environment) to fetch the latest environment variables. None of the environment variables returned matched "".azurewebsites.net" for environment `AzureCloud`. So please correct me if I am wrong - there isnt a `environment().suffixes.<somekey>` that is available for use. Since this current PR is not targeting environments other than AzureCloud, I hope it is ok that I moved the parameter `azureWebsiteDomain` to `variable "azureWebsitesDomain": ".azurewebsites.net",` (change to be submitted soon). Perhaps after this PR, if we target additional azure environments, I will incorporate the `environment()` in if clause and set appropriate value. Does that sound correct?